canonical-webteam/practices
# Add Gemfile and local dev instructions to README

## Done

- Added Gemfile to codify dependencies to run Jekyll site
- Added instructions to README on how to run the site

## QA

- Pull this branch
- Follow README instructions
- Verify you can run site locally
